migrate-profile.md

```diff
@@ -0,0 +1,54 @@
+# Migrate Tanzu Application Platform Profiles
+
+This document describes how to migrate from one Tanzu Application Platform profile to another.
+
+You can perform fresh install of Tanzu Application Platform 1.1 by following the instructions in [Installing Tanzu Application Platform](install-intro.md).
+
+## <a id='prereqs'></a> Prerequisites
+
+Before you migrate from one Tanzu Application Platform to another:
+
+- For information about installing Tanzu Application Platform, see [Install your Tanzu Application Platform profile](install.md#install-profile)
+- For information about installing or updating the Tanzu CLI and plug-ins, see [Install or update the Tanzu CLI and plug-ins](install-tanzu-cli.md#cli-and-plugin)
+- For information on Tanzu Application Platform GUI considerations, see [Tanzu Application Platform GUI Considerations](tap-gui/upgrades.md#considerations)
+- Verify all packages are reconciled by running `tanzu package installed list -A`
+- Note the packages that are included in both the original profile as well as profile being migrated to. If the target profile includes additional packages, those will be added, and additional cluster resources may be needed. However, if the target profile **doesn't include** a particular component, that package will be removed as a result of the migration. If the addition or removal of the packages is of great concern, consider instead adding an additional cluster versus migrating.
+
+## <a id="add-new-package-repo"></a> Add new package repository
+
+Follow these steps to add the new package repository:
+
+1. Add the 1.1.0 version of the Tanzu Application Platform package repository by running:
+
+    ```
+    tanzu package repository update tanzu-tap-repository \
+        --url registry.tanzu.vmware.com/tanzu-application-platform/tap-packages:1.1.0  \
+        --namespace tap-install
+    ```
+
+2. Verify you have added the new package repository by running:
+
+    ```
+    tanzu package repository get tanzu-tap-repository --namespace tap-install
+    ```
+## <a id="edit-profile-values"></a> Edit the tap-values.yaml configuration file that was used during installation
```

Comment:
This is really a generic migration doc. We didn't want to make it targeted to a single use-case. Since Light is deprecated, the idea would be to link to this doc from the former Light Profile section on the install.md mentioning the deprecation and that they should migrate to one of the new profiles.

Reply:
Isnt this migration a one-time thing (eg 1.0 light to 1.1 x) or is it believed folk will regularly do this? Having it cover the latter is adding vagueness to the steps IMHO.

Reply:
The impetus was the Light profile deprecation. However, we can foresee some other circumstances where folks may want to change profile roles (X to Full, Full to Build, etc) so we added some element of generalization.